nflgame/update_players.py

```diff
@@ -181,6 +193,7 @@ def meta_from_soup_row(team, soup_row):
     return {
         'team': team,
         'profile_id': profile_id_from_url(profile_url),
+        'esb_id': esb,
```

Comment:
I initially thought I had it in my code when it was fetching correctly.

Then used: game.players.playerid(pid)

Which fetched the player data from a particular game but was omitting the esb_id.

Working on it now